test_data/config.toml

```diff
@@ -436,468 +263,161 @@ post_cmd = '''
     mv tmp.yaml "$RESPONSE_FILE"
 '''
 
-[search.java_suggestions]
-cmd = "flox search java"
+################################################################################
+# Show
+################################################################################
 
-[show.hello]
-cmd = "flox show hello"
+[show]
+hello.query = "hello"
+flask.query = "python310Packages.flask"
 
-[show.flask]
-cmd = "flox show python310Packages.flask"
+################################################################################
+# Init
+################################################################################
 
 [init.go]
-files = ["init/go/common", "init/go/module"]
-pre_cmd = '''
-    cp -r {common,module}/* .
-'''
-cmd = "flox init --auto-setup"
+unpack_dir_contents = [ "init/go/common", "init/go/module" ]
+auto_setup = true
```

Comment:
suggestion nonblocking: make true the default and drop `auto_setup = true`, looks like we have true for every single case in this section?